src/stores/recipes/recipesReducer.ts

```diff
@@ -43,12 +45,17 @@ export function recipesReducer(
 
       return {
         ...state,
-        recipes,
+        recipes: [...state.recipes, ...recipes],
```

Comment:
1 - when you change sort option, you need to drop recipes only, therefore a separate action is required
2 - when you change filters,  you need to drop recipes only, therefore a separate action is required